client/my-sites/ads/main.jsx

```diff
@@ -112,7 +112,7 @@ class AdsMain extends Component {
 	getComponent( section ) {
 		switch ( section ) {
 			case 'earnings':
-				return <AdsEarnings site={ this.props.site } />;
+				return <WordAdsEarnings site={ this.props.site } />;
```

Comment:
The `site` prop shouldn't be needed as the `WordAdsEarnings` component retrieves the site data from Redux selected site.